Mathlib/Data/Num/Lemmas.lean

```diff
@@ -919,38 +911,37 @@ theorem bitwise'_to_nat {f : Num → Num → Num} {g : Bool → Bool → Bool} (
       repeat'
         rw [show ∀ b n, (pos (PosNum.bit b n) : ℕ) = Nat.bit b ↑n by
           intros b _; cases b <;> rfl]
-      rw [Nat.bitwise'_bit]
-    any_goals assumption
-    any_goals rw [Nat.bitwise'_zero, p11]; cases g true true <;> rfl
-    any_goals rw [Nat.bitwise'_zero_left, this, ← bit_to_nat, p1b]
-    any_goals rw [Nat.bitwise'_zero_right, this, ← bit_to_nat, pb1]
+      rw [Nat.bitwise_bit gff]
+    any_goals rw [Nat.bitwise_zero, p11]; cases g true true <;> rfl
+    any_goals rw [Nat.bitwise_zero_left, ←Bool.cond_eq_ite, this, ← bit_to_nat, p1b]
+    any_goals rw [Nat.bitwise_zero_right, ←Bool.cond_eq_ite, this, ← bit_to_nat, pb1]
     all_goals
-      rw [← show ∀ n : PosNum, ↑(p m n) = Nat.bitwise' g ↑m ↑n from IH]
+      rw [← show ∀ n : PosNum, ↑(p m n) = Nat.bitwise g ↑m ↑n from IH]
       rw [← bit_to_nat, pbb]
-#align num.bitwise_to_nat Num.bitwise'_to_nat
+#align num.bitwise_to_nat Num.bitwise_to_nat
 
 @[simp, norm_cast]
-theorem lor'_to_nat : ∀ m n : Num, ↑(m ||| n) = Nat.lor' m n := by
+theorem lor_to_nat : ∀ m n : Num, ↑(m ||| n) = Nat.lor m n := by
```

Comment:
```suggestion
theorem toNat_lor : ∀ m n : Num, ↑(m ||| n) = Nat.lor m n := by
```
This is just correcting other people's mistakes I know, but we might as well tidy it up, here and everywhere else.

Reply:
I think we should do this in a follow-up PR, where we also fix this to
```suggestion
theorem toNat_lor : ∀ m n : Num, ↑(m ||| n) = m ||| n := by
```

Reply:
okay. I'll let you do the merging, but I approve.